charts/images.yaml

```diff
@@ -86,6 +86,10 @@ images:
   sourceRepository: github.com/gardener/vpn
   repository: eu.gcr.io/gardener-project/gardener/vpn-seed
   tag: "0.19.0"
+- name: vpn-seed-server
+  sourceRepository: github.com/gardener/vpn2
```

Comment:
This repo is still private, please make it at least public, or merge it into the existing `vpn` repo.

Reply:
It's set to public.